README.md

```diff
@@ -106,10 +131,6 @@ bug tracker to voice interest.
 
 Or you can download pre-built SSG zip archive from [latest release](https://github.com/OpenSCAP/scap-security-guide/releases/latest).
```

Comment:
This paragraph is not needed, as there is section  "From release ZIP files ".

Reply:
This paragraph is not being changed in this PR or moved or anything like that. Feel free to make another PR removing it :-)

Reply:
Although it is related to what this PR is about so I have done this change as suggested.